lib/pbench/server/api/resources/query_apis/metadata_index/__init__.py

```diff
@@ -136,3 +137,15 @@ def get_aggregatable_fields(
             for f, v in mappings.get("fields", {}).items():
                 self.get_aggregatable_fields(v, f"{prefix}{f}.", result)
         return result
+
+    def get_mappings(self, document):
```

Comment:
Type hints?  And, a docstring would be nice, since `document` is not exactly self-describing, and knowing that it's a `dict` doesn't help much either.